libs/SalesforceSDKCore/Configuration/SalesforceSDKCore-Common-Dynamic.xcconfig

```diff
@@ -10,6 +10,3 @@
 
 INSTALL_PATH = /
 SKIP_INSTALL = YES
-
-MODULEMAP_FILE = $(SRCROOT)/SalesforceSDKCore/SalesforceSDKCore.modulemap
```

Comment:
The only reason we had a custom module map was for `CSFPrivateDefines.h`. We don't need this anymore.